syntax/elm.vim

```diff
@@ -89,7 +88,7 @@ hi def link elmImport Include
 hi def link elmConditional Conditional
 hi def link elmAlias Delimiter
 hi def link elmOperator Operator
-hi def link elmType Type
+hi def link elmType Identifier
```

Comment:
`elmType` becomes `Identifier` instead of `Type`. Changes like this are a bit suspicious.

Reply:
Thanks for the contribution :)

Reply:
I found this strange as well, I was just keen to try it out and offer it as a draft since I was inspired by #6. I just took a peek how they did it from other fork mention in #6. And that's how they did it. I'm very very new in this syntax thingy.